Functions/Mock.ps1

```diff
@@ -1618,7 +1618,7 @@ function Reset-ConflictingParameters {
     )
 
     $parameters = $BoundParameters.Clone()
-    $names = Get-ConflictingParameterNames
+    $names = $script:ConflictingParameterNames
```

Comment:
As you mentioned in your last comment, using the Get-ConflictingParameterNames functions allows us to mock it during our tests executions.

Please revert this change